test/SubjectTest.fs

```diff
@@ -113,4 +113,61 @@ let tests = testList "Subject Tests" [
         Expect.equal actual1 expected1 "Should be equal"
         Expect.equal actual2 expected2 "Should be equal"
     }
+
+    testAsync "Test replay subject broadcasts last 2 values when second observer subscribes late" {
+        let (dispatch, stream) = AsyncRx.replaySubject 2
+        let obv1 = TestObserver<_>()
+        let obv2 = TestObserver<_>()
+
+        let! _ = stream.SubscribeAsync(obv1)
+
+        do! dispatch.OnNextAsync 'a'
+        do! dispatch.OnNextAsync 'b'
+        do! dispatch.OnNextAsync 'c'
+
+        // TODO: How should I be yielding here so that obv1.Notifications is populated?
+        // (I specifically don't want to emit a completion.) 
+        do! Async.Sleep 1_000
```

Comment:
@dbrattli is there a better way to ensure the value is yielded without using ```Async.Sleep```?

Reply:
I'll have a look, but currently on vacation so might take a few days 😬